invenio_users_resources/records/mappings/v7/users/user-v1.0.0.json

```diff
@@ -69,9 +77,18 @@
           },
           "email_visibility": {
             "type": "keyword"
+          },
+          "locale": {
+            "type": "keyword",
+            "index": "false"
+          },
+          "timezone": {
+            "type": "keyword",
+            "index": "false"
           }
-        }
+        },
+        "dynamic": true
```

Comment:
Does this make all the `preferences` dynamic?

Reply:
my understanding is that all other params not listed (IE those added by the instance) will  be dynamically mapped
@ntarocco am I correct in this?

Reply:
Yes, @alejandromumo see a bit above in `profile` for the same change.